trymito.io/styles/[slug].module.css

```diff
@@ -47,33 +108,48 @@
 
 .post_content h1 {
     margin-top: 3rem;
+    font-size: 2.7rem;
 }
 .post_content h2 {
-    margin-top: 2rem;
+    margin-top: 4rem;
+    font-size: 2rem;
 }
 .post_content h3 {
     margin-top: 2rem;
 }
-.post_content figcaption {
-    color: var(--color-text-secondary);
-    font-size: .7rem;
-    text-align: center;
+.post_content span {
+    font-size: 1.5rem;
+    color: var(--color-text-primary);
+    font-weight: 100;
 }
-
-.post_content blockquote {
-    font-size: 3rem;
-    color: var(--color-text-secondary);
+.post_content code span {
+    font-size: 1rem;
 }
-
 .post_content a {
     color: var(--color-purple);
 }
-
 .post_content a:hover {
     /* Underline */
     text-decoration: underline;
 }
 
+.post_content blockquote {
```

Comment:
There's some wackiness going on here when the screen size gets smaller. This picture is on mobile, but it also happens if you reduce the browser size. 
![IMG_2252](https://github.com/mito-ds/mito/assets/18709905/03afe2a2-a75b-4bcf-b268-4c1448d30839)